src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -343,7 +344,7 @@ and, optionally,
  - `q` [`PhasePartition`](@ref). Without this argument the results are that of dry air.
```

Comment:
In line 327 above,

`internal_energy_sat(ts::PhaseDry{FT}) where {FT<:Real} = cv_m(ts) * (air_temperature(ts) - FT(T_0))`

 I suggest to replace `cv_m(ts)` by `cv_d` (since it is for dry air anyway).